packages/boxel-ui/addon/components/tooltip/index.gts

```diff
@@ -0,0 +1,63 @@
+import Component from '@glimmer/component';
+import { MiddlewareState } from '@floating-ui/dom';
+import { Velcro } from 'ember-velcro';
+import { on } from '@ember/modifier';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+    placement?: MiddlewareState['placement'];
+    offset?: number;
+  };
+  Blocks: {
+    trigger: [];
+    content: [];
+  };
+}
+export default class Tooltip extends Component<Signature> {
+  @tracked isHoverOnHook = false;
+
+  @action
+  onMouseEnter() {
+    this.isHoverOnHook = true;
+  }
+
+  @action
+  onMouseLeave() {
+    this.isHoverOnHook = false;
+  }
+
+  <template>
+    <Velcro @placement={{if @placement @placement 'top'}} @offsetOptions={{if @offset @offset 5}} as |velcro|>
+      <div class='hook' {{velcro.hook}} {{on 'mouseenter' this.onMouseEnter}} {{on 'mouseleave' this.onMouseLeave}}>
+        {{yield to='trigger'}}
+      </div>
+      {{#if this.isHoverOnHook}}
+        <div class='tooltip' {{velcro.loop}}>
+            {{yield to='content'}}
```

Comment:
Indented too much. I suggest to rebase to main and you should get automatic formatting that will take care of this